### PR TITLE
Preparation for v4.3.0

### PR DESCRIPTION
**Pull request by AndreasBilke**

### CODE REVIEW

File: CHANGELOG.txt [hunk 1]
```diff
@@ -2,6 +2,30 @@
 pdfpc Changelog
 ===============
 
+Version 4.3.0
+=============
+
+*Released: December 2018*
+
+- *Breaking changes*
```
Comment (fnevgeny) [inline CHANGELOG.txt]: Backward incompatible changes?

File: CHANGELOG.txt [hunk 2]
```diff
@@ -2,6 +2,30 @@
...
+    - Improvement/more logical default key bindings
+    - GTK+3 >= 3.22
+
+- Four-State operation mode
+    - makes handling of drawing, eraser and pointer mode much easier
+
+- Permament setting of pointer color and size in pdfpcrc
+
+- Window placement can be done by output device name
```
Comment (fnevgeny) [inline CHANGELOG.txt]: output device -> monitor?

**Comment by AndreasBilke** [comment]
@fnevgeny I will merge it and publish a new version when your PR is in. Okay?

**Comment by fnevgeny** [comment]
Sure. Maybe just wait; I might have time to look into #325 over the weekend.

**Comment by AndreasBilke** [comment]
@fnevgeny Is something more in the pipeline or can we merge?

**Comment by fnevgeny** [comment]
I'd say that's it :).

**Pull request merged**